test/unit/create-issue-test.js

```diff
@@ -37,18 +37,51 @@ test('create issue request succeeds', t => {
 
   .then((response) => {
     const createIssueArgs = api.issues.create.lastCall.arg
-    // 1) Make sure the issue was created
     t.is(response.data.html_url, 'html_url')
-    // 2) Make sure it includes the patch, filename,etc
     t.is(createIssueArgs.body, 'test value1: patch value2: filename value3: blobUrl value4: repo')
-    // 3) Make sure the issue was created on the right repo?
     t.is(createIssueArgs.repo, 'repo')
-    // 4) Issue has the correct label?
     t.is(createIssueArgs.labels, 'labels')
-    // 5) That its owner is correct
     t.is(createIssueArgs.owner, 'owner')
 
     simple.restore()
     t.end()
   })
 })
+
+test('create issue request fails', t => {
+  const state = {
+    api,
+    debug: () => {},
+    owner: 'owner',
+    repo: 'repo',
+    branch: 'branch',
+    sha: 'sha',
+    labels: 'labels',
+    commit: {
+      message: 'title',
+      patch: 'patch',
+      filename: 'filename',
+      blobUrl: 'blobUrl'
+    },
+    template: 'test value1: $DIFF value2: $FILENAME value3: $BLOB_URL value4: $REPO'
+  }
+
+  simple.mock(api.issues, 'create').rejectWith({
+    message: 'Not Found',
+    documentation_url: 'https://developer.github.com/v3',
```

Comment:
the message and documentation_url properties are wrong I think, the `error` instance that the `github` API methods reject has a message that looks like `'{"message":"Not Found","documentation_url":"https://developer.github.com/v3"'` and no `documentation_url` property. But also we don’t use them, so you can leave them out, the `code: 404` is enough for this test